Sources/SpeziAccount/Resources/Localizable.xcstrings

```diff
@@ -683,6 +683,9 @@
           }
         }
       }
+    },
+    "Email, Password, Preferences" : {
+
     },
```

Comment:
Not sure if this should have a translation or might be a mistake (or is even fine without a translation)

Reply:
Was there previously but just now got checked due to the rebuild. Was used in a Preview. Changed to code to allow verbatim text in the respective View component.